class ApiError(Exception):
    pass


class AuthError(Exception):
    pass
